extensions/blocks/rating-star/style.scss

```diff
@@ -0,0 +1,101 @@
+// Editor and View styles
+.wp-block-jetpack-rating-star,
+.wp-block-jetpack-rating-spiciness,
+.wp-block-jetpack-rating-priciness {
+	margin-bottom: 1.5em; // Provide a minimum bottom margin.
+	line-height: 0;
+	stroke-width: 0;
+
+	.is-rating-unfilled {
+		fill-opacity: 0.33;
+	}
+
+	.jetpack-ratings-button {
+		display: inline-flex;
+		margin-right: 0.3em;
+
+		// Mimic core focus style.
+		border-radius: 4px;
```

Comment:
Not a biggie and ok as-is, but would you prefer to use Gutenberg core variable here to ensure it stays in sync?

If so, you can copy that variable [from Gutenberg](https://github.com/WordPress/gutenberg/blob/2f68f2885135a4da495b1f94978b26e18a394a31/packages/base-styles/_variables.scss#L67) to [`extensions/shared/styles/gutenberg-variables.scss`](https://github.com/Automattic/jetpack/blob/7e8ff7012b98ae5ff4ced1f5f8915974e4652ab1/extensions/shared/styles/gutenberg-variables.scss) and use it from there.

[I'll replace that file soon](https://github.com/Automattic/jetpack/pull/14092) with core's `base-styles` package so things will just stay in sync without manual effort. 🎉

Reply:
Done at https://github.com/Automattic/jetpack/pull/14091/commits/a54a0d2d8c44d873deafa55b4a1d97fb5e6e1a1f